docs/user/AnsibleCloudAuthentication.md

```diff
@@ -0,0 +1,47 @@
+# Ansible Cloud Provider Authentication
+
+The docker container contains cloud clis needed for interacting with the various clouds.
+
+## GCP
+
+When deploying to GCP we default to using [Google Cloud SQL proxy](https://cloud.google.com/sql/docs/postgres/connect-kubernetes-engine). For security we set this up via workload identity configuration. This requires the following vars to be set:
```

Comment:
Clarify the use case is only required for GCP with external postgres. e.g. 
"When using external postgres in GCP, we default to using..."

"For security the setup is via workload identity configuration"

Reply:
updated